cms/tests/test_health.py

```
@@ -0,0 +1,164 @@
+            response = public_client.get("/health/ready", HTTP_X_FORWARDED_FOR="127.0.0.1")
+        self.assertEqual(response.status_code, 503)
+        body = json.loads(response.content)
+        # Compact shape (string), not dict — proves we did NOT take the privileged branch.
+        self.assertEqual(body["checks"]["database"], "fail")
+        self.assertNotIn("sensitive internal error detail", response.content.decode())
+
+    def test_ready_privileged_response_includes_detail(self):
+        staff = User.objects.create_user(
+            username="staffhealth",
+            email="staffhealth@example.com",
+            password="testpass123",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "password"

(S106)